ukat/data/README.md

```diff
@@ -5,17 +5,17 @@ This document contains two types of information:
 1. Origin of test data
 2. Information which is general enough to apply to all types of MR test data
 
-Specific information for each type of MR data lives on the README.md files in each data subdirectory: [`b0`](b0/README.md), [`dwi`](dwi/README.md), [`r2star`](r2star/README.md).
+Specific information for each type of MR data lives on the README.md files in each data subdirectory: [`b0`](b0/README.md), [`dwi`](dwi/README.md), [`t1`](t1/README.md), [`t2`](t2/README.md) and [`t2star`](t2star/README.md). The data itself is in online storage, we recommend the use of Zenodo and have a [UKRIN community](https://zenodo.org/communities/ukrin/) to help keep data curated. If you want to add data to this community please use [this link](https://zenodo.org/deposit/new?c=ukrin).
```

Comment:
I wonder if this information regarding the data should also be present in the main/parent folder `README.md` file. There's no mention/connection between the links [ukat Zenodo](https://zenodo.org/record/4751354#.YKv8bL5KiUk) and [ukat data Zenodo](https://zenodo.org/communities/ukrin/) - any user should be able to access the "data Zenodo" from the "ukat Zenodo" and vice-versa.

Reply:
Do you mean we should put a link to the Zenodo community in [this readme](https://github.com/UKRIN-MAPS/ukat/blob/master/README.md)? 

I've deliberately kept the two very separate. In theory everyone that uses ukat should be aware of the DOI linked to pypi to reference it. The number of people who will actually contribute to ukat will be a small percent of those who use it, the number of contributors who will want to add data will be an even smaller percent and the number of contributors who want to add data, and want to use Zenodo to host the data rather than their institution web hosting (as is the case for the majority of dataset on [dipy](https://github.com/dipy/dipy/blob/541b5b2992e786b5f51d05649a398cb331903456/dipy/data/fetcher.py#L227)) will be an even smaller subset. I think that adding the link to the community to the main readme would be adding an extra source of confusion for the majority of people to make the life of a couple of contributors slightly easier. 

Worst case if we keep them community links slightly buried, we have to point a couple of people in the right direction on a PR; worst case if the Zenodo community is obvious on the main readme, people use the wrong DOI and thus dilute citations over lots of datasets rather than the single DOI for the code.

Could always add something about it to the contributing guidelines? That way those that might want to be aware it exists *should* have read about it, but a normal user doesn't have to be aware of it.

Reply:
I agree with you, it would indeed more harm than good if it was in the main README.md.

I think a quick mention that it exists in the contributing guidelines would be fine. Something like "Contact the authors if it requires adding data. Examples of datasets can be found in [link]"

Reply:
Cool will add a note to the cotrib guidelines. 

The nifty thing about Zenodo communities is if someone uses the second of the links ([this one](https://zenodo.org/deposit/new?c=ukrin)) they can just drag and drop files and they will be added to Zenodo strait away (and will have a URL etc) so the person uploading can continue writing their code, then once approved the dataset will be added to the UKRIN community. It means people can upload without asking us and then get them added to the community at a later time or not at all if they would rather just keep the dataset separate for whatever reason.